tika-core/src/main/java/org/apache/tika/detect/TSDDetector.java

```diff
@@ -0,0 +1,80 @@
+//package it.notartel.cons.verifyfiles.custom;
+package org.apache.tika.detect;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+
+import org.apache.tika.detect.Detector;
+import org.apache.tika.io.LookaheadInputStream;
+import org.apache.tika.metadata.Metadata;
+import org.apache.tika.mime.MediaType;
+//import org.slf4j.Logger;
+//import org.slf4j.LoggerFactory;
+
+// TSD FILE HEX MAGIC CODE:   30 80 06 0B 2A 86 48 86 F7
+
+public class TSDDetector implements Detector {
+	
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = -3659257718174450902L;
+	
+	//private static final Logger log = LoggerFactory.getLogger(TSDDetector.class);
+	
+	@Override
+	public MediaType detect(InputStream stream, Metadata metadata) throws IOException {
+		
+		final String hexTSDMagicCode = "3080060B2A864886F7";
```

Comment:
I guess, it could be in `static final` field alongside with `byteTSDMagicCode` to avoid extra parsing each time.